backends/bmv2/common/control.cpp

```diff
@@ -352,7 +352,9 @@ ControlConverter::convertTable(const CFG::TableNode* node,
             // 1) if there is at least one RANGE field, then the table is RANGE
             // 2) if there is at least one TERNARY or OPTIONAL field, then the table is TERNARY
             // 3) if there is a LPM field, then the table is LPM
-            // 4) otherwise the table is EXACT
+            // 4) if there is a LPM field, then the table is ternary or EXACT
+            // TODO: Do we also check if table is range or optional in 4?
+            LOG3("Tbl match key: " << match_type << " " << table_match_type);
```

Comment:
This is not a very useful log for developers, you need to indicate which field is which (individual match key element match type vs deducted bmv2 table match type)